spec/css/comment.hrx

```diff
@@ -161,6 +163,55 @@ a {
   b: c e;
 }
 
+<===>
+================================================================================
+<===> sourcemap/options.yml
+:ignore_for:
+  - libsass
+
+<===>
+================================================================================
+<===> sourcemap/sourcemappingurl/input.scss
+a {
+  b: c;
+}
+/*# sourceMappingURL=whatever */
+
+<===> sourcemap/sourcemappingurl/output.css
+a {
+  b: c;
+}
+
+<===>
+================================================================================
+<===> sourcemap/between_imports/input.scss
```

Comment:
"import" is kind of a misleading term here, since this doesn't actually use `@import`. We usually use "load" to refer to "any at-rule that loads another file".